SimPEG/utils/__init__.py

```diff
@@ -104,6 +104,8 @@
 .. autosummary::
   :toctree: generated/
 
+  active_from_xyz
```

Comment:
Not sure what other's opinions are, but mine is that I don't necessarily like importing things defined in `discretize.utils` into `SimPEG.utils` just to shadow them there. I would prefer for the user to import them from `discretize` as needed.

Just trying to say avoid importing *more* things from `discretize` than we already are.

Reply:
I agree that it's weird to import the same things from two places. If it's a discretize function, it should be imported from discretize. And if it's SimPEG functionality, it should be imported from SimPEG.